src/main/java/org/candlepin/subscriptions/tally/job/OfferingSyncConfiguration.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Red Hat, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * Red Hat trademarks are not licensed under GPLv3. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+package org.candlepin.subscriptions.tally.job;
+
+import org.candlepin.subscriptions.spring.JobRunner;
+import org.candlepin.subscriptions.task.queue.TaskProducerConfiguration;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
+import org.springframework.context.annotation.Profile;
+
+/** A class to hold all job related configuration. */
+@Configuration
+@Profile("offering-sync")
+@Import(TaskProducerConfiguration.class)
+@ComponentScan({"org.candlepin.subscriptions.tally.job", "org.candlepin.subscriptions.product"})
```

Comment:
Curious as to why we need to include "org.candlepin.subscriptions.tally.job" as a scan path?

Reply:
Huh! Good point. I guess we don't need it. Deleted. I want to say this configuration class was originally located elsewhere which is why it had it? Maybe. Anyway like I said, it is deleted and tested as still working.